google-play-book-downloader-pdf.py

```diff
@@ -194,11 +194,18 @@ def normalize_windows_cmd_caret(s: str) -> str:
             key, value = header.split(":", 1)
             headers[key.strip()] = value.strip()
 
+    cookies_unparsed = args.cookies
```

Comment:
@coderabbitai fallback to empty array then modify and simplify the rest of the PR accordingly

Reply:
Change `unparsed_cookies` to `cookies_raw`

Reply:
Give me commitable suggestion